docs/installation-and-operation/installing-metabase.md

```diff
@@ -8,21 +8,25 @@ redirect_from:
 
 Metabase is built and packaged as a Java JAR file and can be run anywhere that Java is available.
 
-## [Metabase Cloud](https://store.metabase.com/checkout)
+## Metabase Cloud (Recommended)
```

Comment:
I think this section needs a link to cloud marketing page or docs too, feels a bit weird to direct people who probably just learned about metabase directly to store checkout, without giving them any additional info about what they're signing up for